ext/handlers/filters/messages.go

```diff
@@ -2,6 +2,7 @@ package filters
 
 import (
 	"github.com/PaulSonOfLars/gotgbot/v2"
+        "regexp"
```

Comment:
```suggestion
	"regexp"
	"github.com/PaulSonOfLars/gotgbot/v2"
```

Usual import order is:
- stdlib
- external packages
- local package
with a newline between each category, and each is alphabetically ordered

I appreciate that this isnt clear right now, I'll add some linters to the repo to make sure these are all automatically covered soon.